### PR TITLE
IDEMPIERE-5851 - Set Focus Automatically to the 1st Row of the Info Window Result Set

### PR DESCRIPTION
**Pull request by PeterTakacs300**
JIRA ticket: https://idempiere.atlassian.net/browse/IDEMPIERE-5851

Committing original code by @igorpojzl 

# Pull Request Checklist

- [x] My code follows the [code guidelines](https://wiki.idempiere.org/en/Contributing_to_Trunk) of this project
- [x] My code follows the best practices of this project
- [x] I have performed a self-review of my own code
- [x] My code is easy to understand and review. 
- [x] In hard-to-understand areas, I have commented my code.
- [x] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [ ] I have made corresponding changes to the documentation as follows:
  - [ ] New feature (non-breaking change which adds functionality): I have created the New Feature page in the project wiki explaining the functionality and how to use it. If relevant, I have committed sample data to the core seed to have usable examples in GardenWorld.
  - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected): I have documented the change in a clear way that everyone in the community can understand the impact of the change.
  - [ ] Improvement (improves and existing functionality): This documentation is needed if the improvement changes the way the user interacts with the system or the outcome of a process/task changes. If it is just, for instance, a performance improvement, documentation might not be needed. 
- [ ] The changed/added documentation is in the project wiki (not privately-hosted pdf files or links pointing to a company website) and is complete and self-explanatory.

### CODE REVIEW

**Comment by PeterTakacs300** [comment]
### No multi-selection (1st row automatically selected)
https://github.com/idempiere/idempiere/assets/93127072/ef389e0d-7f9f-4d23-b7ac-4a182c271234

### Multi-selection (1st row focused but not selected)
https://github.com/idempiere/idempiere/assets/93127072/fd300e61-cf6b-458d-9523-2dbadd5b8c1a

**Comment by PeterTakacs300** [comment]
Hi @idempiere/core, could I ask you to check this PR? It would help us, if you could find the time. It's a relatively small improvement and has been opened for a while.

Also as PR https://github.com/idempiere/idempiere/pull/1999, these are improvements that has been in our system for a while and had a positive feedback from our customers, we thought they could be useful in the community, wdyt?

**Comment by hengsin** [comment]
[pr2001.patch.txt](https://github.com/idempiere/idempiere/files/13195008/pr2001.patch.txt)

- please review and test, thanks.

**Comment by PeterTakacs300** [comment]
Thanks @hengsin, patch tested successfully.

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/editor/WTableDirEditor.java [hunk 1]
```diff
@@ -259,7 +259,7 @@ private void init()
     {
     	ZKUpdateUtil.setWidth(getComponent(), "200px"); 
         getComponent().setAutocomplete(true);
-        getComponent().setAutodrop(false);
+        getComponent().setAutodrop(true);
```
Comment (hengsin) [inline org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/editor/WTableDirEditor.java:262]: this change is not related to this ticket, please remove it.

**Review by hengsin — CHANGES_REQUESTED** [review]

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/editor/WTableDirEditor.java (hunk 1 above)
Comment (PeterTakacs300) [reply org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/editor/WTableDirEditor.java]: ok, done

**Review by hengsin — APPROVED** [review]
tested
